BUILD.md

```diff
@@ -251,7 +251,13 @@ These commands assume working [Go](https://golang.org) and
     bin/web setup
     ```
 
-1. Install Linkerd on a Kubernetes cluster.
+2. Fetch the necessary Go dependencies:
```

Comment:
TIOLI: you might want to say `install JS dependencies` on line 247, so it's parallel with the `Go dependencies` in this line